src/sympc/module/nn/conv.py

```diff
@@ -140,11 +141,25 @@ def share_state_dict(
 
         self.kernel_size = (kernel_size_w, kernel_size_h)
         self.weight = MPCTensor(secret=weight, session=self.session, shape=shape)
+        self._parameters = OrderedDict({"weight": self.weight})
 
         if bias is not None:
             self.bias = MPCTensor(
                 secret=bias, session=self.session, shape=(self.out_channels,)
             )
+            self._parameters["bias"] = self.bias
+
+    def parameters(self, recurse: bool = False) -> MPCTensor:
```

Comment:
Q: Does this solves the None issue in step while training?

Reply:
No, this doesn't. The whole GradDiv PR solves that issue. Basically, MSELoss has a division operation. GradDiv hasn't defined some gradients after the division was None.